README.md

```diff
@@ -186,6 +187,57 @@ In this example response, aggregated time series information for a range of minu
 This information has as its origin the 46nd minute, of the 2nd hour of February, the 18th, 2015. And includes data for the
 13th second, for which there is a sample and the sum (and value of that sample) is 34.59.
 
+On the other hand, if the attribute value was of type string, a query such as the following (with `aggrMethod` as `occur`)
+sent to the STH component:
+
+<pre>http://localhost:8666/STH/v1/contextEntities/type/&lt;entityType&gt;/id/&lt;entityId&gt;/attributes/&lt;attrName&gt;?aggrMethod=occur&aggrPeriod=second&dateFrom=2015-02-22T00:00:00.000Z&dateTo=2015-02-22T23:00:00.000Z</pre>
+
+may end up receiving the following payload as a possible response:
+
+<pre>
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "attrName",
+                        "values": [
+                            {
+                                "_id": {
+                                    "origin": "2015-02-18T02:46:00.000Z",
+                                    "range": "minute",
+                                    "resolution": "second"
+                                },
+                                "points": [
+                                    {
+                                        "offset": 35,
+                                        "samples": 34,
+                                        "occur": {
+                                            "string01": 7,
```

Comment:
That's right... :) It just counts the occurrences of the values assigned to certain `string` attribute over time... This `string` value can be anything... :) Any `string` value, I mean... :)